src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,31 +1,56 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import deepGet from '../../helpers/json/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import getPassportHome from '../../lib/utilities/getPassportHome';
 
 const logger = nodeLogger(__filename);
 
-// checks for data and status keys, setting default status if not found
+// checks for data, status, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
 });
 
-// checks for pageData and 200 status
+const validatePassportHome = passportHome => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { service } = useContext(ServiceContext);
+
+  return passportHome ? passportHome === service : true;
+};
+
+// checks for pageData, 200 status and if home service from article data fits the service locale
 const shouldRender = data => {
   const { status, pageData } = constructRenderObject(data);
 
+  let statusCode = status;
+  let isValidHome;
+
   const hasDataAnd200Status = pageData && status === 200;
+  if (hasDataAnd200Status) {
+    const passportHome = pageData && getPassportHome(pageData);
```

Comment:
```suggestion
    const passportHome = getPassportHome(pageData);
```
This isn't needed since we already check for pageData in the if block surrounding this.